docs/modules/ppo.rst

```diff
@@ -31,6 +31,14 @@ Notes
 Can I use?
 ----------
 
+.. note::
+
+  A recurrent version of PPO is available in our contrib repo: https://sb3-contrib.readthedocs.io/en/master/modules/ppo_recurrent.html
+
+  However we advise users to start with simple frame-stacking as a simpler, faster
+  and usually competitive alternative, more info in our report: https://wandb.ai/sb3/no-vel-envs/reports/PPO-vs-RecurrentPPO-aka-PPO-LSTM-on-environments-with-masked-velocity--VmlldzoxOTI4NjE4
```

Comment:
Could also point to Procgen paper's appendix with similar results (Fig 11): https://arxiv.org/abs/1912.01588

Also maybe a mention that you can do framestacking with `VecFrameStack`?